### PR TITLE
AMP-77114 - update TS Browser web attribution 

### PR DESCRIPTION
**Pull request by Mercy811**
# Amplitude Developer Docs PR

## Description

https://amplitude.atlassian.net/browse/AMP-77114
1. Fix web attribution versions for browser SDKs 
image
2. Add `attribution` and `defaultTracking` to configuration
image
3. Add `config.attribution.trackNewCampaigns`
4. Add note of priority of `config.defaultTracking.pageViews` and `config.attribution.trackPageViews` to answer https://amplitude.atlassian.net/browse/AMP-77114

## Deadline

When do these changes need to be live on the site?

## Change type

- [ ] Doc bug fix. Fixes #[insert issue number]. Amplitude contributors include Jira issue number. 
- [ ] Doc update.
- [ ] New documentation.
- [ ] Non-documentation related fix or update.

# PR checklist:

- [ ] My documentation follows the style guidelines of this project.
- [ ] I previewed my documentation on a local server using `mkdocs serve`.
- [ ] Running `mkdocs serve` didn't generate any failures.
- [ ] I have performed a self-review of my own documentation.

@amplitude-dev-docs

### CODE REVIEW

File: docs/data/sdks/typescript-browser/migration.md [hunk 1]
```diff
@@ -542,9 +542,9 @@ For `amplitude-js`, two separate callback functions are passed for success and e
...
 | Default Event Tracking [^2] |  [**Default Event Tracking V2**](./#default-event-tracking-v2-vs-default-event-tracking-v1) | [**Default Event Tracking V1**](./#default-event-tracking-v2-vs-default-event-tracking-v1) |  Not supported. |
-| Configuration | Configuration is implemented by Configuration object during initialize amplitude. [More configurations](../#configuration). Check [here](./) for migration guide from the Maintenance SDK to the latest SDK. | The same as latest Browser SDK.  | Support explicity setter methods. [More configurations](../../javascript/#configuration). |
+| Configuration | Configuration is implemented by Configuration object during initialize amplitude. [More configurations](../#configuration). Check [here](./) for migration guide from the Maintenance SDK to the latest SDK. | The same as latest Browser SDK.  | Support explicitly  setter methods. [More configurations](../../javascript/#configuration). |
```
Comment (justin-fiedler) [inline docs/data/sdks/typescript-browser/migration.md:547]:
```suggestion
| Configuration | Configuration is implemented by Configuration object during initialize amplitude. [More configurations](../#configuration). Check [here](./) for migration guide from the Maintenance SDK to the latest SDK. | The same as latest Browser SDK.  | Support explicit  setter methods. [More configurations](../../javascript/#configuration). |
```

File: docs/data/sdks/typescript-browser/index.md [hunk 2]
```diff
@@ -34,17 +34,20 @@ Use [this quickstart guide](../sdk-quickstart#browser) to get started with Ampli
...
-    |`attribution.initialEmptyValue`| `string`. Customize the initial empty value for attribution related user properties to any string value. | `EMPTY` |
-    |`attribution.trackNewCampaigns`| `boolean`. Whether tracking new campaigns on the current session. | `false` | 
-    |`attribution.trackPageViews`| `boolean`. Whether track page view on attribution. | `false` |
+    |`config.attribution.disabled`| `boolean`. Whether disable the attribution tracking. | `false` |
+    |`config.attribution.excludeReferrers`| `string[]`. Exclude the attribution tracking for the provided referrers string | Including all referrers by default. |
+    |`config.attribution.initialEmptyValue`| `string`. Customize the initial empty value for attribution related user properties to any string value. | `EMPTY` |
+    |`config.attribution.resetSessionOnNewCampaign`| `boolean`. Whether to reset user sessions when a new campaign is detected. Note a new| `false` |
+    |`config.attribution.trackNewCampaigns`| `boolean`. Whether tracking new campaigns on the current session. | `false` | 
+    |`config.attribution.trackPageViews`| `boolean`. Whether track page view on attribution. | `false` |
```
Comment (justin-fiedler) [inline docs/data/sdks/typescript-browser/index.md:50]: Maybe we should add a callout here about DET taking precedence and link to the section below.

**Review by justin-fiedler — APPROVED** [review]
Thanks @Mercy811 looks good. See nits.

File: docs/data/sdks/typescript-browser/index.md (hunk 2 above)
Comment (yuhao900914) [reply docs/data/sdks/typescript-browser/index.md:50]: Why we added `config.` here?

**Review by yuhao900914 — APPROVED** [review]

File: docs/data/sdks/typescript-browser/index.md (hunk 2 above)
Comment (Mercy811) [reply docs/data/sdks/typescript-browser/index.md]: Hi @yuhao900914, I feel like adding `config` here can make the hierarchy more clear. Also added `config` to `config.defaultTracking`

**Pull request merged**